index.js

```diff
@@ -114,6 +115,11 @@ exports.init = function (Implementation) {
             directorySmartImplementation = path.resolve('.') + '/forest';
           }
 
+          if(directorySmartImplementation){
+            inspectCodeSyntax
+            .extractCodeSyntaxErrorInDirectoryFile(directorySmartImplementation);
```

Comment:
You need to indent the second line to show this is the continuation of instruction beginning at line 119.